crates/partition-store/src/keys.rs

```diff
@@ -553,6 +553,10 @@ impl KeyCodec for TimerKeyKind {
                 let invocation_uuid = InvocationUuid::decode(source)?;
                 TimerKeyKind::CleanInvocationStatus { invocation_uuid }
             }
+            3 => {
+                let invocation_uuid = InvocationUuid::decode(source)?;
+                TimerKeyKind::NeoInvoke { invocation_uuid }
+            }
```

Comment:
It seems that we are lacking test coverage for the `NeoInvoke`. Can we add it?

Reply:
Added